### DISCUSSION TITLE
Suggestion, list opencascade.js on OpenCascade.org's projects and products page

### DISCUSSION THREAD

**Discussion by Irev-Dev** [donalffons/opencascade.js#63] [Ideas]
https://dev.opencascade.org/about/projects_and_products

There's a "request to list your project" button.

**Comment by donalffons** [comment]
Nice, that's a great idea! I didn't know that page.
I will apply to be listed there. Thanks for sharing!

**Reply by Irev-Dev** [reply]
You're welcome :)

It's a recent initiative from OpenCascade so you're not late to the party or anything.

**Reply by donalffons** [reply]
...finally managed to fill out OpenCascade's questionnaire. This project is now [listed](https://dev.opencascade.org/project/opencascadejs) on their site. Thanks again!

**Reply by Irev-Dev** [reply]
Awesome!

**Reply by Irev-Dev** [reply]
Maybe I should have made this an issue so it could be closed 🤷
